### PR TITLE
Completed the catalog hovers task

### PR DESCRIPTION
**Pull request by Shevtsooov**
- [DEMO LINK](https://Shevtsooov.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/blocks/catalog.scss [hunk 1]
```diff
@@ -17,9 +17,15 @@
...
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
   }
+
+  &__card:hover {
+    transform: scale(120%);
+    color: #34568b;
+  }
```
Comment (serentinos) [inline src/styles/blocks/catalog.scss]: You can nest hover inside card style. 
```
&__card {
  &:hover {
    transform: scale(120%);
    color: #34568b;
  }
}
```
Also better use the coefficient for scale(), not percents I guess.

[checklist p3](https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md)

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: $hover-transition;
   }
+
+  &__link:hover {
+    color: #00acdc;
+  }
```
Comment (serentinos) [inline src/styles/blocks/header.scss]: Same here
[checklist p3](https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md)

**Review by serentinos — CHANGES_REQUESTED** [review]
Good job! Here are a few suggestions

**Comment by Shevtsooov** [comment]
@serentinos Thank you! Corrected the nesting.

**Review by andrusb21 — APPROVED** [review]
Good Job! 🔥🔥🔥

File: src/styles/blocks/catalog.scss [hunk 3]
```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```
Comment (silchyn) [inline src/styles/blocks/catalog.scss:20]: so why you don't use your variable here?

Comment (silchyn) [inline src/styles/blocks/catalog.scss:20]: only add transition for needed properties

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**